model/openapi.yml

```diff
@@ -197,7 +205,8 @@ components:
           description: The ApiKey token that fleet-server has generated for the enrolling agent.
           type: string
           format: password
-        status: # FIXME Does this have any value? The UI does other calculations to infer status.
+        status:
+          deprecated: true
```

Comment:
I can't comment on the exact line, but while you are in here can you add `starting` as a state the agent can report to fleet?

https://github.com/elastic/fleet-server/blob/6e7ed9ebb20679c1463a1e8effd6c281d4f845d9/model/openapi.yml#L226-L233

Reply:
added